pkg/mkimage-raw-efi/install

```diff
@@ -208,14 +267,23 @@ if mount_part INVENTORY "$(root_dev)" -t vfat -o iocharset=iso8859-1; then
 fi
 
 # we also maybe asked to pause after
-grep -q eve_pause_after_install /proc/cmdline && pause "before shutting the node down"
+grep -q eve_pause_after_install /proc/cmdline && pause "shutting the node down"
 
 # lets hope this is enough to flush the caches
 sync; sleep 5; sync
 for p in CONFIG INVENTORY P3; do
    umount "/run/$p" 2>/dev/null
 done
 
+if [ "$POOL_CREATION_COMMAND" != "" ] && [ $DISKS_TO_MERGE_COUNT -gt 1 ]; then
```

Comment:
Why is this delayed to this point?

Reply:
Because we want to have `collect_black_box` before unmount.

Reply:
got it!